tensorpack/libinfo.py

```diff
@@ -63,4 +63,4 @@
 # These lines will be programatically read/write by setup.py
 # Don't touch them.
 __version__ = '0.9.0.1'
-__git_version__ = __version__
+__git_version__ = "v0.0.0-87-g3ac448a-dirty"
```

Comment:
Is this intentional?

Reply:
> Is this intentional?

This might be introduced from patch. I changed it back